src/data_gradients/feature_extractors/classification/class_distribution.py

```diff
@@ -0,0 +1,79 @@
+import collections
+
+import pandas as pd
+
+from data_gradients.common.registry.registry import register_feature_extractor
+from data_gradients.feature_extractors.abstract_feature_extractor import Feature
+from data_gradients.utils.data_classes.data_samples import ClassificationSample
+from data_gradients.visualize.seaborn_renderer import BarPlotOptions
+from data_gradients.feature_extractors.abstract_feature_extractor import AbstractFeatureExtractor
+
+
+@register_feature_extractor()
+class ClassificationClassDistribution(AbstractFeatureExtractor):
```

Comment:
The name is not consistent with Segmentation and Detection. (`SegmentationClassFrequency` and `DetectionClassFrequency`)

Reply:
But I also notice that the variables shown are different.
Here you show the count, and in the seg/detection it's the frequency.
We moved from count to frequency because it helps to compare train/val sets distribution gap

Reply:
Fixed in 8530cb21cca2851d3fadfdc0f57f82362db43853